requirements/dev.txt

```diff
@@ -1,9 +1,16 @@
 # Dependencies needed only for development/testing.
 
+coverage==5.0.3 \
+    --hash=sha256:d649dc0bcace6fcdb446ae02b98798a856593b19b637c1b9af8edadf2b150bea
+
+pytest-cov==2.8.1 \
+    --hash=sha256:cdbdef4f870408ebdbfeb44e63e07eb18bb4619fae852f6e760645fa36172626
+
 pytest==5.3.5 \
     --hash=sha256:ff615c761e25eb25df19edddc0b970302d2a9091fbce0e7213298d85fb61fef6 \
     --hash=sha256:0d5fe9189a148acc3c3eb2ac8e1ac0742cb7618c084f3d228baaec0c254b318d
 
+
```

Comment:
Please remove this extra line.

Reply:
Oops. Drawbacks of using Vim! Let me rectify it

Reply:
@armenzg just a little doubt. How do you push the code to repo after 2F authentication is enabled?
Created my personal access token. But it's throwing error 403